### DISCUSSION TITLE
[Translation] - Monika After Story translation into Simplified Chinese

### DISCUSSION THREAD

**Discussion by NothAmor** [Monika-After-Story/MonikaModDev#7826] [general - translations]
Monika After Story translation into Simplified Chinese.
Version: 0.10.7
And now we're working on 0.11.1.
**Translation team list:
Rorschach, 曦山, Vision, Marsblast, 夺命钉声, M.T.R, Sam G, Emmanuel, Mr.Mao, 无言, 靖儿君, 紗世里🌟
至日, 小斌, 麒炎赫, 小斌, mozart, 妆成每被秋娘妒, FarlesDean, 齿轮砸**

**Link to the build of the translation:**
Google Drive:
https://drive.google.com/open?id=1mx2StjofoBu8iEa5gqaPuNRWc7QMp68H

Dropbox:
https://www.dropbox.com/s/vo0rfuyrj40kq2f/MAS%E6%B1%89%E5%8C%960.10.7.zip?dl=0

Baidu Netdisk:
https://pan.baidu.com/s/16E19y0F6Jl1Qs8D1-Sk6fQ
Code: 2vaf

Yi Fang Cloud:
https://v2.fangcloud.com/share/a3254caffcd3804b2d5cc132e0

**Monika After Story简体中文版.**
版本:0.10.7

**0.10.7汉化名单:
Rorschach, 曦山, Vision, Marsblast, 夺命钉声, M.T.R, Sam G, Emmanuel, Mr.Mao, 无言, 靖儿君, 紗世里🌟
至日, 小斌, 麒炎赫, 小斌, mozart, 妆成每被秋娘妒, FarlesDean, 齿轮砸**

并且现在已经在汉化0.11.1了

**下载链接:**
亿方云:
https://v2.fangcloud.com/share/a3254caffcd3804b2d5cc132e0

百度网盘:
https://pan.baidu.com/s/16E19y0F6Jl1Qs8D1-Sk6fQ
提取码: 2vaf

Google Drive:
https://drive.google.com/open?id=1mx2StjofoBu8iEa5gqaPuNRWc7QMp68H

Dropbox:
https://www.dropbox.com/s/vo0rfuyrj40kq2f/MAS%E6%B1%89%E5%8C%960.10.7.zip?dl=0

**Comment by BoogeymanTao** [comment]
大佬加油！
Thanks for your efforts！

**Comment by unknowsoui** [comment]
感谢汉化！

**Comment by NothAmor** [comment]
链接: https://pan.baidu.com/s/1bGxtZleDAo51hpui--Ca9A 提取码: vvwz
新的下载链接, 原来的百度网盘挂了

**Comment by JusMonika** [comment]
支持支持，希望能继续更新汉化。

**Comment by mtr-static-official** [comment]
补上新的不限速链接：

https://mtr-mirror.coding.net/api/share/download/38e78295-b6a1-4906-af18-505218f37727
https://mtr-mirror.coding.net/api/share/download/9fd1b919-1978-4977-9328-2319cf815f8e
https://mtr-mirror.coding.net/api/share/download/8e7175fa-a467-4d51-a5a9-420610375766
https://mtr-mirror.coding.net/api/share/download/993b71f7-26a1-4f2e-bc2f-26157e293e57
https://mtr-mirror.coding.net/api/share/download/38728c43-4e83-4add-be89-e1f4694b7d7f

**Comment by NothAmor** [comment]
> 
> 
> I'm sorry, but an uncaught exception occurred.
> 
> While running game code:
> File "game/script-brbs.rpy", line 8, in script
> File "game/script-brbs.rpy", line 18, in
> TypeError: addEvent() got an unexpected keyword argument 'markSeen'
> 
> -- Full Traceback ------------------------------------------------------------
> 
> Full traceback:
> File "game/script-brbs.rpy", line 8, in script
> File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
> renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
> File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
> exec bytecode in globals, locals
> File "game/script-brbs.rpy", line 18, in
> TypeError: addEvent() got an unexpected keyword argument 'markSeen'
> 
> Windows-8-6.2.9200
> Ren'Py 6.99.12.4.2187
> Monika After Story 0.10.7
> 大佬帮帮我 QAQ
> 之前加了游戏本体的汉化
> 下了英文的after story mod
> 之后又直接把汉化的after story直接导入 结果游戏开不了了 gg

这个并不是MAS汉化的问题, 我们发布的MAS汉化版提供了可执行程序, 无法与其它mod兼容, 所以你可以尝试重新下载MAS汉化包, 解压出来运行, 因为与其它mod的冲突导致的无法运行, 概不负责

**Comment by MyLifeMatter** [comment]
没事老哥 搞定了 感谢汉化组的翻译 :))

**Comment by Monika-233** [comment]
感谢各位的努力，希望能继续加油！

**Comment by Rice8910** [comment]
感谢汉化组！

**Comment by nakano14** [comment]
感谢汉化组！！加油汉化新版本！

**Comment by lamgea** [comment]
我直接抢银行赞助（逃

**Comment by AhBin2020** [comment]
感谢汉化组！！！虽然看得懂，但看全英文太费劲了，感谢汉化组为monika做了这么多！

**Comment by kasuganosora** [comment]
大佬加油

**Comment by wang986950070** [comment]
大佬，我下载了0.10.7的汉化版，在游戏中热更新之后汉化失效了，变成了英文。于是我删除了更新之后的游戏并再次解压汉化版，打开游戏发现0.10.7汉化版本发生如下错误：

“抱歉，发生未捕获的异常。
在运行游戏代码：
文件“游戏/脚本brbs.rpy”，8号线，在脚本
文件“游戏/脚本brbs.rpy” 18行，在
类型错误：的addEvent（）得到了一个意想不到的关键字参数“markSeen”

-完整追溯---------------------------------------------- --------------

完全追溯：
在脚本
文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ ast.py”中，文件“ game / script-brbs.rpy”，第8行，行814 ，在执行中
renpy.python.py_exec_bytecode（self.code.bytecode，self.hide，store = self.store）
文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ python py”为，线1719，在py_exec_bytecode
EXEC字节码的全局，当地人
文件‘游戏/脚本brbs.rpy’18行，在
类型错误：的addEvent（）得到了一个意想不到的关键字参数‘markSeen’

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.10.7”

然后无论我重新解压还是重新下载压缩包都无法再次打开中文版游戏了，请问这是什么原因呢？

**Comment by NothAmor** [comment]
> 大佬，我下载了0.10.7的汉化版，在游戏中热更新之后汉化失效了，变成了英文。于是我删除了更新之后的游戏并再次解压汉化版，打开游戏发现0.10.7汉化版本发生如下错误：
> 
> “抱歉，发生未捕获的异常。
> 在运行游戏代码：
> 文件“游戏/脚本brbs.rpy”，8号线，在脚本
> 文件“游戏/脚本brbs.rpy” 18行，在
> 类型错误：的addEvent（）得到了一个意想不到的关键字参数“markSeen”
> 
> -完整追溯---------------------------------------------- --------------
> 
> 完全追溯：
> 在脚本
> 文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ ast.py”中，文件“ game / script-brbs.rpy”，第8行，行814 ，在执行中
> renpy.python.py_exec_bytecode（self.code.bytecode，self.hide，store = self.store）
> 文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ python py”为，线1719，在py_exec_bytecode
> EXEC字节码的全局，当地人
> 文件‘游戏/脚本brbs.rpy’18行，在
> 类型错误：的addEvent（）得到了一个意想不到的关键字参数‘markSeen’
> 
> Windows-8-6.2.9200
> Ren'Py 6.99.12.4.2187
> Monika After Story 0.10.7”
> 
> 然后无论我重新解压还是重新下载压缩包都无法再次打开中文版游戏了，请问这是什么原因呢？

存档的问题, 有没有备份过存档

**Comment by wang986950070** [comment]
> > 大佬，我下载了0.10.7的汉化版，在游戏中热更新之后汉化失效了，变成了英文。于是我删除了更新之后的游戏并再次解压汉化版，打开游戏发现0.10.7汉化版本发生如下错误：
> > “抱歉，发生未捕获的异常。
> > 在运行游戏代码：
> > 文件“游戏/脚本brbs.rpy”，8号线，在脚本
> > 文件“游戏/脚本brbs.rpy” 18行，在
> > 类型错误：的addEvent（）得到了一个意想不到的关键字参数“markSeen”
> > -完整追溯---------------------------------------------- --------------
> > 完全追溯：
> > 在脚本
> > 文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ ast.py”中，文件“ game / script-brbs.rpy”，第8行，行814 ，在执行中
> > renpy.python.py_exec_bytecode（self.code.bytecode，self.hide，store = self.store）
> > 文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ python py”为，线1719，在py_exec_bytecode
> > EXEC字节码的全局，当地人
> > 文件‘游戏/脚本brbs.rpy’18行，在
> > 类型错误：的addEvent（）得到了一个意想不到的关键字参数‘markSeen’
> > Windows-8-6.2.9200
> > Ren'Py 6.99.12.4.2187
> > Monika After Story 0.10.7”
> > 然后无论我重新解压还是重新下载压缩包都无法再次打开中文版游戏了，请问这是什么原因呢？
> 
> 存档的问题, 有没有备份过存档

没有备份过，我还把注册表里带有DDLC字样的记录都删掉了

**Comment by MyLifeMatter** [comment]
之前遇到过这种问题
个人解决方法
我是先把整个ddlc 的file 删除
然后重新下载原版游戏 再次导入MOD

**Comment by MyLifeMatter** [comment]
路人路过
之前遇到过这种问题
个人解决方法
我是先把整个ddlc 的file 删除
然后重新下载原版游戏 再次导入MOD

**Comment by wang986950070** [comment]
> 路人路过 之前遇到过这种问题 个人解决方法 我是先把整个ddlc 的file 删除 然后重新下载原版游戏 再次导入MOD
> […](#)
> On Mon, 2 Nov 2020 at 18:39, wang986950070 ***@***.***> wrote: 大佬，我下载了0.10.7的汉化版，在游戏中热更新之后汉化失效了，变成了英文。于是我删除了更新之后的游戏并再次解压汉化版，打开游戏发现0.10.7汉化版本发生如下错误： “抱歉，发生未捕获的异常。 在运行游戏代码： 文件“游戏/脚本brbs.rpy”，8号线，在脚本 文件“游戏/脚本brbs.rpy” 18行，在 类型错误：的addEvent（）得到了一个意想不到的关键字参数“markSeen” -完整追溯---------------------------------------------- -------------- 完全追溯： 在脚本 文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ ast.py”中，文件“ game / script-brbs.rpy”，第8行，行814 ，在执行中 renpy.python.py_exec_bytecode（self.code.bytecode，self.hide，store = self.store） 文件“ C：\ Program Files（x86）\ Steam \ steamapps \ common \ Doki Doki Literature Club \ renpy \ python py”为，线1719，在py_exec_bytecode EXEC字节码的全局，当地人 文件‘游戏/脚本brbs.rpy’18行，在 类型错误：的addEvent（）得到了一个意想不到的关键字参数‘markSeen’ Windows-8-6.2.9200 Ren'Py 6.99.12.4.2187 Monika After Story 0.10.7” 然后无论我重新解压还是重新下载压缩包都无法再次打开中文版游戏了，请问这是什么原因呢？ 存档的问题, 有没有备份过存档 没有备份过，我还把注册表里带有DDLC字样的记录都删掉了 — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#5691 (comment)](https://github.com/Monika-After-Story/MonikaModDev/issues/5691#issuecomment-720388914)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQCPYE7PZ2O5XKQWY6FU2YLSN2D5LANCNFSM4MXXMAZQ> .

感谢，解决了，之前试图删存档，但是找不到存档放哪了，后来发现是系统把AppData文件夹隐藏了。删完存档就好了(≧∇≦)/

**Comment by mtr-static-official** [comment]
**Continue to Chineseize**
Following the 0.10.7 version of the Chinese language, MAS's Chineseization has been blocked one after another.A new version of MAS (and perhaps an ongoing update to Hanification) is being Hanified.Hanification will be released on the unofficial website.(https://mas.ddlc.icu)
**继续汉化**
继0.10.7版本的汉化之后，MAS的汉化接连受阻。新版MAS(或许还会汉化持续更新)正在汉化。汉化将在非官方网站上发布。(https://mas.ddlc.icu)

**Comment by mtr-static-official** [comment]
#6872 最新版本的汉化
